### PR TITLE
Add Release Notes for iOS Release 1.68.134

### PR DESCRIPTION
**Pull request by Uni-verse**
Resolves https://github.com/brave/brave-browser/issues/41013

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:
N/A

### CODE REVIEW

File: CHANGELOG_iOS.md [hunk 1]
```diff
@@ -1,22 +1,58 @@
 # Changelog
 
+## [1.68.134](https://github.com/brave/brave-browser/releases/tag/v1.68.134)
+
+### Web3
+
+ - Added the ability to import BTC accounts (BIP84) into Brave Wallet. ([#38546](https://github.com/brave/brave-browser/issues/38546))
+ - Added the ability to send BTC on “Send” screen. ([#37958](https://github.com/brave/brave-browser/issues/37958))
```
Comment (srirambv) [inline CHANGELOG_iOS.md:8]: Update to  `Added support to "Send" BTC from iOS.`

File: CHANGELOG_iOS.md [hunk 2]
```diff
@@ -1,22 +1,58 @@
...
+ - Updated default account names to differentiate between mainnet and testnet accounts. ([#38526](https://github.com/brave/brave-browser/issues/38526))
```
Comment (srirambv) [inline CHANGELOG_iOS.md:9]: Specific change to Filecoin so better to add that 
`Updated default account names to differentiate between Filecoin mainnet and testnet accounts.`

File: CHANGELOG_iOS.md [hunk 3]
```diff
@@ -1,22 +1,58 @@
...
+ - Updated “Show Zero Balances” button to always be displayed in “Select a Token to Send” screen. ([#38452](https://github.com/brave/brave-browser/issues/38452))
```
Comment (srirambv) [inline CHANGELOG_iOS.md:10]: Maybe change to 
`Updated “Show Zero Balances” button to be always shown on "Select a Token to Send” screen.`

**Review by srirambv — CHANGES_REQUESTED** [review]
Once change its a good to go for Web3

File: CHANGELOG_iOS.md [hunk 4]
```diff
@@ -1,22 +1,58 @@
...
+ - Added ability to use quick actions via "/" under Brave Leo panel. ([#37297](https://github.com/brave/brave-browser/issues/37297))
+ - Added support for conversation API in Leo. ([#39331](https://github.com/brave/brave-browser/issues/39331))
+ - Added UI views to handle Premium disconnected states in Leo chat. ([#39231](https://github.com/brave/brave-browser/issues/39231))
+ - Added name indicator beside the avatar for each message in the Leo chat. ([#37793](https://github.com/brave/brave-browser/issues/37793))
+ - Added the ability to close all tabs except active tab using “Close All Other Tabs” in the tab tray context menu. ([#37069](https://github.com/brave/brave-browser/issues/37069))
+ - Implemented blocking non-http(s) URLs from loading when opened via Shortcuts. ([#39836](https://github.com/brave/brave-browser/issues/39836))
+ - Implemented displaying Sync code expiry when showing the pairing codes. ([#38020](https://github.com/brave/brave-browser/issues/38020))
+ - Updated ads service preferences to disable fetching when Brave Reward/Brave News is disabled. ([#39621](https://github.com/brave/brave-browser/issues/39621))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:26]: nit: `Brave Reward` --> `Brave Rewards`

**Comment by Uni-verse** [comment]
> Once change its a good to go for Web3

Thank you for reviewing @srirambv 

https://github.com/brave/brave-browser/commit/b08f0f8e85586a9effd8a1972d9223f00dae9332

File: CHANGELOG_iOS.md [hunk 5]
```diff
@@ -1,22 +1,58 @@
...
+ - Updated displaying items on History page by handling certain title cases. ([#39550](https://github.com/brave/brave-browser/issues/39550))
+ - Updated settings icon design in the Leo chat header. ([#39460](https://github.com/brave/brave-browser/issues/39460))
+ - Updated the design for the Leo opt-in notice. ([#38742](https://github.com/brave/brave-browser/issues/38742))
+ - Updated Leo chat prompt to add new line instead of submitting the message when pressing “enter” key. ([#39351](https://github.com/brave/brave-browser/issues/39351))
+ - Updated the title in the Leo chat product header to “Leo AI”. ([#36917](https://github.com/brave/brave-browser/issues/36917))
+ - Fixed issue with Leo chat window being unresponsive after using app switcher while the Leo menu is open. ([#39109](https://github.com/brave/brave-browser/issues/39109))
+ - Fixed buttons not working to dismiss the Premium upgrade card in Leo chat. ([#39061](https://github.com/brave/brave-browser/issues/39061))
```
[the remark this answers is not in the document]
Comment (Uni-verse) [reply CHANGELOG_iOS.md:33]: Yeah thanks, I'm also not certain. You're probably right.

**Comment by Uni-verse** [comment]
> There's a few nit but generally looks good 👍

Thanks for reviewing!
https://github.com/brave/brave-browser/commit/8354fbe77b5942c1eb00beda467e8812a67ac596

**Review by kylehickinson — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
LGMT 👍

**Review by srirambv — APPROVED** [review]
++

**Pull request merged**